phpstan.neon

```diff
@@ -17,6 +17,8 @@ parameters:
         - '#Call to an undefined method Prophecy\\Prophecy\\ObjectProphecy::[a-zA-Z0-9_]+\(\)#'
         - '#Method ApiPlatform\\Core\\Tests\\Bridge\\Doctrine\\Orm\\ItemDataProviderTest::getManagerRegistry\(\) should return Doctrine\\Common\\Persistence\\ManagerRegistry but returns object\.#'
         - '#Method ApiPlatform\\Core\\Tests\\Bridge\\Doctrine\\Orm\\Util\\IdentifierManagerTraitTest::getObjectManager\(\) should return Doctrine\\Common\\Persistence\\ObjectManager but returns object\.#'
+        - '#Parameter \#[0-9] \$offset of function array_splice expects int, null given\.#'
```

Comment:
This looks dangerous, but it's definitely a phpstan error, it comes from https://github.com/api-platform/core/blob/master/src/GraphQl/Serializer/ItemNormalizer.php#L82 but our value can't be null, as stated in the docs:

> As with all internal PHP functions as of 5.3.0, array_search() returns NULL if invalid parameters are passed to it.

But our arguments are always correct. Is it because of a bad return type of the `getAllowedAttributes` method? 
I've tried to change the code a bit to test for null values but it doesn't fix the issue either.

Reply:
Hum, not sure... The `AbstractNormalizer::getAllowedAttributes()` definition says `@return string[]|AttributeMetadataInterface[]|bool`. So, the executed code can potentially be `array_search('id', false, true)` which will return `null`.